addons/controllers/clusterbootstrap_controller.go

```diff
@@ -1149,6 +1150,12 @@ func (r *ClusterBootstrapReconciler) GetDataValueSecretNameFromBootstrapPackage(
 			if err = r.Get(r.context, key, secret); err != nil {
 				r.Log.Error(err, "unable to fetch secret for package with inline config", "objectkey", key)
 				return "", err
+			} else {
+				// secret for package with inline does not exist, we should create one
+				_, err := r.cbHelper.CreateSecretFromInline(cluster, cbPkg, packageRefName)
```

Comment:
@shivaani0505 Did you test this in a cluster and worked well? Why this change takes care of updating inline values?

An alternative would be to do inline processing in reconcileNormal right after the call to ```createOrPatchClusterBootstrapFromTemplate()``` (for both create and update).

Reply:
Updated code to address the highlighted issue.